src/CheckoutSdk.Microsoft.Extensions/CheckoutOptions.cs

```diff
@@ -6,13 +6,13 @@ public class CheckoutOptions
     {
         public string SecretKey { get; set; }
         public string PublicKey { get; set; }
-        public bool Sandbox { get; set; }
+        public bool? Sandbox { get; set; }
         public string Uri { get; set; }
 
         public CheckoutConfiguration CreateConfiguration()
         {
-            var checkoutConfiguration = string.IsNullOrEmpty(Uri)
-                ? new CheckoutConfiguration(SecretKey, Sandbox)
+            var checkoutConfiguration = string.IsNullOrWhiteSpace(Uri)
+                ? new CheckoutConfiguration(SecretKey, Sandbox ?? true)
```

Comment:
Why not have your bool as Production, that way the default is sandbox and you don’t need to have a nullable book?

Reply:
This is the standard we agreed for all the SDKs